FWCore/Integration/test/unit_test_outputs/testSubProcess.grep.txt

```diff
@@ -12,3 +12,4 @@ got data of type "edmtest::Doodad" with name "abcd" in record GadgetRcd
 ++++++++ finished: processing esmodule: label = '' type = DoodadESSource in record = GadgetRcd
 got data of type "edmtest::Doodad" with name "abc" in record GadgetRcd
 got data of type "edmtest::Doodad" with name "abc" in record GadgetRcd
+   25 Tracer               -s DoodadESSource:D                       4        4
```

Comment:
@makortel
"25" should be a "24" in 12_4_X (whatever it means...)

Reply:
@perrotta Thanks for the suggestion, it indeed works with 12_4_X.   A new fix is made: #38767 38767